HTTPServerProject/HTTPServerProxy/Response.cs

```diff
@@ -0,0 +1,135 @@
+using HTTPServerRead.Streams;
+using HTTPServerRead.Header;
+using HTTPServerRead.Body;
+using HTTPServerWrite.Response;
+using HTTPServerWrite.Streams;
+
+namespace HTTPServerProxy.Response;
+
+public class ProxyResponse
+{
+    Header _header;
+    Body _body;
+    IReadStreams _reader;
+    IWriteStreams _writer;
+    string _path;
+    string _type;
+
+    public ProxyResponse(IReadStreams reader, IWriteStreams writer, string path, string type)
+    {
+        _reader = reader;
+        _writer = writer;
+        _header = new Header(_reader);
+        _body = new Body(_reader);
+        _path = path;
+        _type = type;
+    }
+
+    public void GetResponse()
```

Comment:
I mentioned this in the tests - but `GetResponse` implies this returns a response so it should have another name.

Reply:
Agreed.